test/format/value.test.ts

```diff
@@ -444,6 +480,24 @@ describe('format', function () {
                     },
                 },
             );
+            io.set(
+                {$type: 'yql.timestamp64', $value: '-62167305600000000', $optional: 2},
+                {
+                    json: {
+                        plain: '-000002-12-31T00:00:00.000000Z',
+                        html: '<span class="yql_timestamp64">-000002-12-31T00:00:00.000000Z</span>',
+                    },
+                },
+            );
+            io.set(
+                {$type: 'yql.timestamp64', $value: '-62167305600000000', $optional: 0},
+                {
+                    json: {
+                        plain: '-000002-12-31T00:00:00.000000Z',
+                        html: '<span class="yql_timestamp64">-000002-12-31T00:00:00.000000Z</span>',
+                    },
+                },
+            );
```

Comment:
It looks like the changes affects the following types:
- `yql.date`
- `yql.date32`
- `yql.datetime`
- `yql.datetime64`
- `yql.timestamp`

At the moment unit-tests added only for `yql.data 32`, `yql.datetime64` and `yql.timestamp 64`. Isn't it better to add tests for each affected type?